scripts/check-helper.sh

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env bash
+
+set -x # Show commands
+set -eu # Errors/undefined vars are fatal
+set -o pipefail # Check all commands in a pipeline
+
+if [[ $# -ne 4 ]]
+then
+    set +x  # Turn off echoing of commands and output only relevant things to avoid bloating logfiles
```

Comment:
A little unsure that you need the `set +x` here, since in general this path shouldn't get hit much in practice and if it does spewing one extra "echo blah blah" command isn't the end of the world. But I don't mind leaving it here.

Reply:
I mainly just copied and pasted the existing "set +x" boilerplate to clarify what was happening and in case it got copied onward.  The main intent here was to avoid the echo command line being confused for its output, or rather, distracting the eye (which happened for me).  So I'll leave as-is with the consensus that log filling isn't a real problem for these.